src/components/attendee/Password/ProfileModalChangePassword.module.scss

```diff
@@ -1,11 +1,13 @@
+@use "scss/attendee/space";
+
 .ProfileModalChangePassword {
   &__input {
-    margin: 8px 0;
+    margin: space.$sm 0;
```

Comment:
Please use `space.gap()` and `space.button()`, just like `space.empty()` to add semantic meaning / uniformity. Wouldn't be against of adding one more `space` function if we need for some common thing

Reply:
[84cb042](https://github.com/sparkletown/sparkle/pull/3030/commits/84cb042903c42e21a52cb03d0e89e0b6aed4cdcb)